src/benchmark_register.cc

```diff
@@ -293,6 +305,16 @@ Benchmark* Benchmark::Ranges(const std::vector<std::pair<int, int>>& ranges) {
   return this;
 }
 
+Benchmark* Benchmark::ArgName(const std::string& name) {
+  arg_names_ = {name};
+  return this;
+}
+
+Benchmark* Benchmark::ArgNames(const std::vector<std::string>& names) {
+  arg_names_ = names;
```

Comment:
Oh, I see now that there are checks in `Arg/Args/Range/Ranges/DenseRange` that use `ArgsCnt`. Previously I thought that you could call `Args` with different number of elements each time.
I'll add these checks then, it seems to be the way to go.